frankenphp/docker-entrypoint.sh

```diff
@@ -56,8 +56,18 @@ if [ "$1" = 'frankenphp' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ]; then
 	php bin/console lexik:jwt:generate-keypair || true
 	php bin/console app:update-rdap-servers
 
-	setfacl -R -m u:www-data:rwX -m u:"$(whoami)":rwX var
-	setfacl -dR -m u:www-data:rwX -m u:"$(whoami)":rwX var
+	SETFACL_STATUS=0
+	SETFACL_ERROR=$(setfacl -m u:www-data:rwX -m u:"$(whoami)":rwX var 2>&1 1>/dev/null) || SETFACL_STATUS=$?
+	if [ $SETFACL_STATUS -eq 0 ]; then
```

Comment:
This can be simplified
```suggestion
	if setfacl -m u:www-data:rwX -m u:"$(whoami)":rwX var 2>/dev/null; then
```

And the error message is printed to STDERR only, there is no need to redirect STDOUT to /dev/null.